fluidreview/management/commands/importfluidusers.py

```diff
@@ -0,0 +1,34 @@
+"""
+Command for importing FluidReview users
+"""
+from urllib.parse import urlparse
+
+from django.core.management import BaseCommand
+
+from fluidreview import serializers
+from fluidreview.api import FluidReviewAPI, process_user
+
+
+class Command(BaseCommand):
+    """
+    Create user and profile objects for each FluidReview user
+    """
+    def handle(self, *args, **options):
+        """
+        Run the command
+        """
+        api = FluidReviewAPI()
+        url = 'users'
+        while url:
```

Comment:
Is it possible to abstract out the code as a generator? I'm thinking something like

    for fluid_user in api.list_users():
        serializer = serializers.UserSerializer(data=fluid_user)
        if serializer.is_valid(raise_exception=False):
            process_user(serializer.data)
            ...

Reply:
sure